charmcraft/commands/store/charmlibs.py

```diff
@@ -42,9 +51,9 @@ class LibInternals:
     lib_id: str
     api: int
     patch: int
-    pydeps: list
+    pydeps: List[str]
     content_hash: str
-    content: bytes
+    content: str
```

Comment:
is the type change here on purpose? why?

Reply:
It is — when I was making these changes I double-checked it with mypy and discovered it was always supposed to be a `str`